docs/guides/getting-started-java-spring.md

```diff
@@ -36,18 +29,26 @@ For example, in this guide we will outline a BPMN model to receive a payment req
 While stepping through this guide, you can visit our [sample repository](https://github.com/camunda/camunda-8-get-started-spring/blob/main/src/main/java/io/camunda/demo/process_payments/ChargeCreditCardWorker.java) with the completed code to check your work.
 :::
 
-<SmPrereqs/>
+## Prerequisites
 
-## Step 1: Install Camunda 8 Self-Managed
+Before getting started, ensure you have the following in your local environment:
 
-<Install/>
+- Access to your preferred code editor or IDE
+- [OpenJDK 21+](https://openjdk.org/install/)
+- [Camunda 8 Desktop Modeler](https://camunda.com/download/modeler/) and [Camunda 8 Run<br/>](https://camunda.com/download/modeler/)_If you've already signed up for a Self-Managed account, you can find the download links in your welcome email._
```

Comment:
I noticed that C8Run is linked to the modeller link. 
Could we add a link to C8Run installation page? 
I would not expect users to wait for email or look for email when they are already committed to trying out this guide. 
Plus, we do not get the download link in our email. It is not as easy as downloading an app. it needs a few more steps.

Reply:
It looks like the download emailer hasn't gone up yet. I'll add the link to the C8 Page.